### ISSUE TITLE
Add support for Unix Shell Files

### ISSUE THREAD

**Issue by jfisbein** [Glavin001/atom-beautify#580]
Atom Beautify could not determine a supported beautifier to handle this file with grammar "Shell Script" and extension "sh".

Thanks!!!

**Comment by quilicicf** [comment]
+1, would be really helpful !

**Comment by Glavin001** [comment]
Waiting on Pull Request to be merged: https://github.com/Glavin001/atom-beautify/pull/1220

**Comment by Glavin001** [comment]
Sounds like a duplicate of #451.
